.storybook/themes/commerce.scss

```diff
@@ -301,6 +313,102 @@
 					gap: var(--global-spacing-2),
 					width: var(--content-scale-width),
 				),
+				date: (
+					font-size: var(--body-font-size-small),
+					line-height: var(--body-line-height-small),
+				),
+				heading: (
+					font-weight: var(--global-font-weight-7),
+				),
+				icon: (
+					fill: var(--icon-fill-color),
+					height: var(--global-spacing-5),
+					width: var(--global-spacing-5),
+				),
+				input: (
+					align-items: flex-start,
+					display: flex,
+					flex-direction: column,
+					gap: var(--global-spacing-1),
+				),
+				input-input: (
+					border: var(--global-border-width-1) var(--global-border-style-1) var(--form-border-color),
+					border-radius: var(--global-border-radius-1),
+					padding: var(--global-spacing-2) var(--global-spacing-4),
+				),
+				input-input-focus: (
+					outline: solid 2px var(--form-border-color),
+					outline-offset: 1px,
+				),
+				input-label: (
+					font-weight: var(--global-font-weight-7),
+				),
+				input-error-input: (
+					border-color: var(--form-border-color-danger),
+				),
+				input-error-input-focus: (
+					outline-color: var(--form-border-color-danger),
+				),
+				input-success-input: (
+					border-color: var(--form-border-color-success),
+				),
+				input-success-input-focus: (
+					outline-color: var(--form-border-color-success),
+				),
+				input-warning-input: (
+					border-color: var(--form-border-color-warning),
+				),
+				input-warning-input-focus: (
+					outline-color: var(--form-border-color-warning),
+				),
+				link: (
+					color: var(--color-primary),
+				),
+				media-item: (
+					align-items: flex-start,
+					display: flex,
+					flex-direction: column,
+					gap: var(--global-spacing-1),
+				),
+				media-item-fig-caption: (
+					color: var(--body-text-color-subtle),
+					font-size: var(--body-font-size-tiny),
+					line-height: var(--body-line-height-tiny),
+				),
+				media-item-title: (
+					font-weight: var(--global-font-weight-7),
+				),
+				overline: (
+					font-size: var(--global-font-size-7),
+					font-weight: var(--global-font-weight-7),
+					line-height: var(--global-line-height-4),
+				),
+				paragraph: (
+					font-family: var(--font-family-secondary),
+				),
+				pill: (
+					background-color: var(--color-primary),
+					border-radius: 2em,
+					color: var(--global-white),
+					font-size: var(--global-font-size-2),
+					line-height: var(--global-line-height-2),
```

Comment:
**Note:** `pill` is an exception to its typographic specs in Figma. It will use global tokens for `font-size` and `line-height`. Typography will be the same on all breakpoints.